src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -94,13 +94,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
 
       // get convo info 
-      List<Conversation> conversations = conversationStore.getAllConversations();
-      newUsers.add((User)request.getAttribute("newUser") );
+      List<Message> messages = messageStore.getLatestMessages();
 
       // recieve new convo from convo servlet
```

Comment:
receive

Reply:
Could you please clarify what I should do here?